score.yaml

```diff
@@ -44,6 +44,7 @@ run:
     default: {conda: xpdsim}
   xpdtools:
     default: {conda: xpdtools}
+    bleeding: {orch: 'https://github.com/xpdAcq/xpdtools.git'}
```

Comment:
what is this?

Reply:
This allows us to build both off the conda package and the bleeding edge code. It helps with keeping things consistent across releases.